### PR TITLE
feat: add dark/light switch

### PR DESCRIPTION
**Pull request by pedronauck**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1,40 @@
...
+
+# Pinlock to exact version (default is '^')
+# https://pnpm.io/npmrc#save-prefix
+# see also how save-workspace-protocol affect this https://pnpm.io/npmrc#save-workspace-protocol
+save-prefix=''
+
+# Most of the time, you want to use the rolling protocol for monorepos
+# https://pnpm.io/npmrc#save-workspace-protocol
+save-workspace-protocol=rolling
+
```
Comment (luizstacio) [inline .npmrc:38]: Why we need this on this PR? If is a general improvement split into another PR

Comment (pedronauck) [reply .npmrc]: 🙄

**Review by LuizAsFight — APPROVED** [review]

**Review by luizstacio — APPROVED** [review]
